feed-integration-webapp/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/crater_atomfeed
```

Comment:
default application properties are good idea, however without a way to override them it will make things difficult for someone to change - even for other properties file that may require dynamic property. 

You should consider externalizing configuration .. use a propertysourceconfigurer, use @value('property-name') and leverage using -D parameter, or system variable or even  either pass at startup "--spring.config.location=classpath:/another-location.properties"

The above leaves options open and does not require code change and allows docker or other environments to manage them externally and pass them to the app using whatever means suits them